dask_ml/model_selection/_incremental.py

```diff
@@ -715,16 +796,33 @@ class IncrementalSearch(BaseIncrementalSearch):
     ...           'l1_ratio': np.linspace(0, 1, num=1000),
     ...           'average': [True, False]}
 
-    >>> search = IncrementalSearch(model, params, random_state=0)
+    >>> search = IncrementalSearchCV(model, params, random_state=0)
     >>> search.fit(X, y, classes=[0, 1])
-    IncrementalSearch(...)
+    IncrementalSearchCV(...)
 
     Alternatively you can provide keywords to start with more hyper-parameters,
     but stop those that don't seem to improve with more data.
 
-    >>> search = IncrementalSearch(model, params, random_state=0,
-    ...                            n_initial_parameters=1000,
-    ...                            patience=20, max_iter=100)
+    >>> search = IncrementalSearchCV(model, params, random_state=0,
+    ...                              n_initial_parameters=1000,
+    ...                              patience=20, max_iter=100)
+
+    ``tol`` and ``patience`` are for stopping training on a "plateau", or when
+    the hold-out score flattens and/or falls. For example, setting ``tol=0``
+    and ``patience=2`` will dictate that scores are increasing (except for
+    potentially the last score).
```

Comment:
Should the parenthetical be "except for potentially when `max_iter` is reached"? That's a bit more descriptive.

Reply:
I was covering that when `patience=2` and `tol=0`, only the last score is decreasing if the model does plateau. I tried to clarify that with:

> ``tol`` and ``patience`` are for stopping training on a "plateau", or when
the hold-out score flattens and/or falls. For example, setting ``tol=0``
and ``patience=2`` will dictate that for estimators that never plateau,
scores will always increase until ``max_iter`` calls are reached.
If it the estimator plateaus, only the last score will decrease.

Let me know what edits you have.